data-and-programming.Rmd

```diff
@@ -53,7 +53,7 @@ If you wish to use `<-`, you will still need to use `=`, however only for argume
 
 Because vectors must contain elements that are all the same type, `R` will automatically coerce to a single type when attempting to create a vector that combines multiple types.
 
-```{r}
+```{r results = 'hold'}
```

Comment:
I think @daviddalpiaz isn't a fan of this approach. I've tried to push _inlining_ the output without code separations. c.f. 

https://twitter.com/axiomsofxyz/status/992828971728408576

Reply:
See my comment on the PR. I just made this connect and now kind of like it. Still thinking about it though.

Reply:
I think your observation that it better matches what happens in RStudio is very prescient.   Looking at html output, the "stack of boxes" that you get breaks the flow  enough to be distracting. (For me anyway.)  If you really want to show a break, you should probably be using multiple code blocks and putting some explanation (such as text, or a leading comment) between the blocks.